src/kbmod/work_unit.py

```diff
@@ -78,11 +78,13 @@ def __init__(
         per_image_indices=None,
         lazy=False,
         file_paths=None,
+        obstimes=None,
```

Comment:
I would recommend updating the docstring too.  If you have good context for `lazy` and `file_paths` that would be nice too.